### ISSUE TITLE
A couple of suggestions

### ISSUE THREAD

**Issue by joshterrill** [brinkmanjg/ng2-typeahead#22]
1. When clicking on the input, could there be a flag to where it expands and shows all the options? And then you can start typing from there to filter down?

2. In the example when you load the page, it starts with Apple selected, then you click it and it says "Begin typing a fruit", and then when you click off of it, the palceholder text still says "Begin typing a fruit". I feel like it should go to be something else, like the state it was at before I clicked it?

**Comment by techhysahil** [comment]
That's right, i am also looking for same things. I will try to add it and send the pull request.
